runners/gearpump/src/main/java/org/apache/beam/runners/gearpump/translators/GearpumpPipelineTranslator.java

```diff
@@ -19,11 +19,9 @@
 package org.apache.beam.runners.gearpump.translators;
 
 import com.google.common.collect.ImmutableList;
-
```

Comment:
Any idea why we have so such noise in the diff ? Maybe your eclipse changed the formatting ? It would be great to fix that to have a clean diff only on the actual checkstyle update.